blog/_posts/2018-07-09-IntroducingTeamRubies.md

```diff
@@ -0,0 +1,123 @@
+
+Atibhi is a 3rd year Undergrad student who thinks she is Monica Geller's real life version. She has lost over 30kgs and can be found reading or coding if she is not worrying about how to become fitter. She is happiest when travelling. She makes sure she knows the historical background of a new place before she visits it and can spend her nights at museums.
+An electronics student by day and open source contributor at night, she hopes to someday combine her love for hardware and software !
+
+Prateksha is a 2nd year Undergrad student, who can be found humming a tune or dancing to one if she isn't coding. Solving algorithms keeps her engaged in her free time. She loves shopping and exploring new places. She enjoys meeting new people and interacting with them. It amuses her to know about different cultures and ideologies. Apart from this, she loves organizing and volunteering at events.
+
+### **Finding Coaches**
+
+They had found each other but now they had to find coaches. They found four awesome coaches.
+
+**Aditya T** - Aditya is their senior at college. He is specialising in Pattern Recognition and Signal Processing. He is presently an Artificial Intelligence developer for Klarity Law and did his GsoC 2017 with Systers.
+
+**Athitya** - To the outside world, he's a Mechie at IIT,Kharagpur. But, with the help of his friends at IITKGP, he codes. He's a wizard with Ruby and did his GSoC 2017 at Ruby Foundation. He is interning at Intuit this summer.
+
+**Gaurav** - Gaurav is their super senior at college who interned at Microsoft this summer. He is a coding enthusiast and a proficient web developer. He works on how Machine Perception can help us reach from Artificial Narrow Intelligence to Artificial General Intelligence. He did his GSoC 2017 at Berkman Centre.
+
+**Raounaq** - Raounaq is a passionate software developer who uses Ruby to express in code what defines him. Apart from coding he played for a football club from Delhi, India. He also likes to travel.
+
+**Brihi** - Atibhi's childhood friend, RGSoC 2017 scholar and a person who loves Code, Coffee and Music. Brihi can work with anything that is associated with Data, Machine Learning, and AI. Even though she isn't their official coach, RGSoC 2018 wouldn't have been possible without her. From reviewing their application to jumping with joy when they got selected, she has been their biggest supporter throughout the process !
+
+### **Choosing a project and interacting with the mentors !**
+
+They had chosen their project even before the list of official projects had been announced. They were thrilled when their primary choice "if-me" got selected for RGSoC'18.
+They could relate to "if-me" and wanted to contribute to the mental health community. Here is their first contact with Julia, their mentor way before the official list was announced.
+
+![FirstContact](/img/blog/2018/2018-07-09-TeamRubies-FirstContact.png)
```

Comment:
Please add pic credits like other images for this photo/screenshot [in the text].